Makefile

```diff
@@ -26,13 +26,19 @@ debug:
 debug-run:
 	HUMAN_LOG=1 DEBUG=1 go run -tags 'debug' $(LDFLAGS) main.go
 
+.PHONY: lint-local
+lint-local:
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.50.1
+	golangci-lint run ./...
+
 .PHONY: lint
-lint:
-	exit
+lint: generate-prod
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.50.1
+	golangci-lint run ./...
 
-.PHONY: test
+generate-prod:
```

Comment:
Is this supposed to be a new block?

Reply:
I think this may be a merge issue

Reply:
My mistake ive added .PHONY: test which was missing.

Reply:
sorry, one more very minor issue: could you also remove the 'generate-prod' line, please? (I think it's not needed, and there isn't any generate-prod step either)